source/agora/consensus/protocol/Nominator.d

```diff
@@ -75,6 +76,12 @@ public extern (C++) class Nominator : SCPDriver
     /// Whether we're in the process of nominating
     private bool is_nominating;
 
+    // values used to switch hash function between priority and neighborhood checks
+    // which are from SCPDriver.cpp
+    const uint hash_N = 1;
+    const uint hash_P = 2;
+    const uint hash_K = 3;
```

Comment:
Please move these inside of `computeHashNode`. Constants should only be in an outer scope if they're used by other code too, but they are not used anywhere else.